config/samples/backends/pure/nvme-tcp/backend.yaml

```diff
@@ -0,0 +1,32 @@
+# To be able to use this sample it is necessary to:
+# - Have a Pure backend with iSCSI support
+# - Have the Pure storage credentials in cinder-volume-pure-secrets.yaml
+# - Having iscsid and multpathd running on the host (done by this sample)
```

Comment:
iscsid not required for NVMe

Reply:
True, it requires the `nvme-fabrics` kernel module